parm/parm_gdas/atm_crtm_coeff.yaml

```
@@ -0,0 +1,178 @@
+mkdir:
+- $(DATA)/crtm/
```

```suggestion
- $(DATA)/crtm
```
I don't think a trailing `/` is necessary.

---

Change committed at c27618bb